lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -56,7 +59,7 @@ class AxeAudit extends Audit {
       extendedInfo: {
         value: rule,
       },
-      details: Audit.makeTableDetails(headings, items),
+      details: {...Audit.makeTableDetails(headings, items), impact, tags},
```

Comment:
should `impact` and `tags` be nested or something? I don't really have an opinion, this seems fine and nesting might not be any better, but it may be putting us on a path toward `details` becoming the new grab bag :)

Reply:
FWIW, I like it where it is. 

If it were nested, I can't think of what it would be called. On Deque's site "impact" and "tags" meta is extrapolated to create the "[Compliance Data & User Impact](https://dequeuniversity.com/rules/axe/2.2/color-contrast?application=lighthouse)" column. "complianceUserImpact:" seems a little long anyway ;-)